precovery/frame_db.py

```diff
@@ -547,7 +561,14 @@ def load_hdf5(
             chunksize=chunksize,
         ):
             observations = [Observation.from_srcobs(o) for o in src_frame.observations]
-            data_uri, offset, length = self.store_observations(observations)
+            year_month_str = "-".join(
+                Time(src_frame.mjd, format="mjd", scale="utc").isot.split("-")[:2]
+            )
```

Comment:
This feels like it deserves a more prominent billing, maybe as a method on the Frame class. Otherwise this is the only spot where it resides.

Reply:
Agreed. Calendar-month from mjd is also implemented in `adam-etl` already as well. This would benefit from being somewhere in our new theoretical `adam.core`.